apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -110,7 +110,7 @@ div.pl-order-blocks-container {
 }
 
 div.pl-order-blocks-header {
-  height: 3rem;
+  height: 5rem;
```

Comment:
Is there a need for a fixed height at all? Why not just allow it to expand/contract based on its content?

Reply:
Agreed, better to just let it expand and contract itself based on content and screen size.

Reply:
That's a good point, I've adjusted it so that it should expand/contract based on the content: 
<img width="300" alt="Capture d’écran, le 2024-10-01 à 23 07 00" src="https://github.com/user-attachments/assets/c077e8db-c830-445a-ad1f-ad88da2edd12">